tests/snapshots/tests.contrib.graphql.test_graphql.test_graphql_error.json

```diff
@@ -0,0 +1,51 @@
+[[
+  {
+    "name": "graphql.query",
+    "service": "graphql",
+    "resource": "{ invalid_schema }",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "web",
+    "meta": {
+      "runtime-id": "f9b0ea34d35f48f4890e4da8a182484f"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.measured": 1,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "system.pid": 51171
+    },
+    "duration": 975000,
+    "start": 1652688948723253000
+  },
+     {
+       "name": "graphql.parse",
+       "service": "graphql",
+       "resource": "graphql.parse",
+       "trace_id": 0,
+       "span_id": 2,
+       "parent_id": 1,
+       "type": "web",
+       "metrics": {
+         "_dd.measured": 1
+       },
+       "duration": 110000,
+       "start": 1652688948723380000
+     },
+     {
```

Comment:
I did not add error tags to this PR. This query failed during the validate step and returned before `graphql.execute()` was called. Is this acceptable?

Reply:
We should be flagging the span as an error with all the error information we can to help the user debug the app.